csm_web/frontend/src/components/Course.js

```diff
@@ -42,7 +43,8 @@ export default class Course extends React.Component {
       currDayGroup: "",
       showUnavailable: true,
       userIsCoordinator: false,
-      showModal: false
+      showModal: false,
+      studentData: false // !!!!!!! SET TO FALSE LATER !!!!!!s
```

Comment:
Is this just distinguishing between whether the modal should be exporting student data or creating a section? It feels very weird to have this as global state; at the very least it would be better as an enum (e.g. modalKind = "exportCsv" | "createSection") rather than just a boolean.